index.html

```diff
@@ -46,37 +46,37 @@ <h1 class="hero-h1">Travel</h1>
                     <h2 class="hero-h2"> Who Travels?</h2>
                     <p class="hero-body">It could be you! We're the one stop shop for all time and space travel needs.</p>
                 </span>
+                 <div class="entire-form-content">
+                     <div class="hero-body form-title">
+                         <p>Explore a place in time or space</p>
+                     </div>
+                     <div>
+                         <form>
+                             <div class="search-form">
+                                 <div class="search-form-item">
+                                     <label for="form-destination">Destination:</label><br>
+                                     <input type="text" id="form-destination" name="form-destination">
+                                 </div>
+                                 <div class="search-form-item">
+                                     <label for="form-time">Time Period:</label><br>
+                                     <input type="text" id="form-time" name="time-period">
+                                 </div>
+                                 <div class="search-form-item">
+                                     <label for="form-time">Travel Dates:</label><br>
+                                     <input type="date" id="form-dates" name="travel-dates">
+                                 </div>
+                             </div>
+                         </form>
+                          <div class="search-form-button">
```

Comment:
It's because its working on a two row single column grid. I did this for simplicity sake to make sure the code was clean, since I was having trouble trying to work with the button and form side by side -- I couldn't get the widths of the columns with percentages or fr. So now the button and form are together in a div. 
<img width="1101" alt="Screen Shot 2021-12-14 at 6 56 16 PM" src="https://user-images.githubusercontent.com/93295768/146103102-6915c1ca-b818-4c4e-b90b-0da8d1362f6c.png">

Reply:
Cool. Just wanted to make sure this was intended. Might it be possible to use a text-align on the button's class to align it to the right?  It makes for a more logical flow.

Reply:
Let me try, because I agree. It would look better to the right.

Reply:
Perfect, it worked! I see what I did wrong now -- I was calling the class with the button, instead of just the class. Button should be right aligned now.